README.md

```diff
@@ -123,152 +125,244 @@ Congratulations! You can now send remote push notifications to your device from
+config.urlDelegate = YourCustomUrlDelegate()
+config.customActionDelegate = YourCustomActionDelegate()
+```
+	
+###### Changing the display interval between in-app messages
+To customize the time delay between successive in-app messages (default value of 30 seconds), set `IterableConfig.inAppDisplayInterval` to an appropriate value (in seconds). 
+
+##### 4. Tracking Custom Events
+Custom events can be tracked using `IterableAPI.track(event:...)` calls.
 	
-5. ##### Updating User Fields
-	User fields can be modified using `IterableAPI.updateUser` call. You also have `updateEmail` and `updateSubscriptions` methods.
+##### 5. Updating User Fields
+User fields can be modified using `IterableAPI.updateUser` call. You also have `updateEmail` and `updateSubscriptions` methods.
 	
-6. ##### Disabling Push Notifications to a Device
-	When a user logs out, you typically want to disable push notifications to that user/device. This can be accomplished by calling `disableDeviceForCurrentUser`. Please note that it will only attempt to disable the device if you have previously called `registerToken`.
+##### 6. Disabling Push Notifications to a Device
+When a user logs out, you typically want to disable push notifications to that user/device. This can be accomplished by calling `disableDeviceForCurrentUser`. Please note that it will only attempt to disable the device if you have previously called `registerToken`.
 	
-	In order to re-enable push notifcations to that device, simply call `registerToken` as usual when the user logs back in.
+In order to re-enable push notifcations to that device, simply call `registerToken` as usual when the user logs back in.
 	
-7. ##### Uninstall Tracking
-	Iterable will track uninstalls with no additional work by you. 
+##### 7. Uninstall Tracking
+Iterable will track uninstalls with no additional work by you. 
 
-	This is implemented by sending a second push notification some time (currently, twelve hours) after the original campaign. If we receive feedback that the device's token is no longer valid, we assign an uninstall to the device, attributing it to the most recent campaign within twelve hours. A "real" campaign send (as opposed to the later "ghost" send) can also trigger recording an uninstall. In this case, if there was no previous campaign within the attribution period, an uninstall will still be tracked, but it will not be attributed to any campaign.
+This is implemented by sending a second push notification some time (currently, twelve hours) after the original campaign. If we receive feedback that the device's token is no longer valid, we assign an uninstall to the device, attributing it to the most recent campaign within twelve hours. A "real" campaign send (as opposed to the later "ghost" send) can also trigger recording an uninstall. In this case, if there was no previous campaign within the attribution period, an uninstall will still be tracked, but it will not be attributed to any campaign.
```

Comment:
- ~we receive~ the app receives
- ~we assign~ it assigns
- can also trigger the 

Suggestion:

> To do this, Iterable sends a silent push notification some time (currently, twelve hours) after a campaign has been sent. Based on this silent push notification, if Iterable receives feedback that the device token is no longer valid, it assigns an uninstall to the device based on the prior campaign. Similarly, if a "real" campaign uncovers an invalid device token, it will also check for a prior (within twelve hours) campaign to mark as the cause for the uninstall. If there was no recent campaign, Iterable still tracks the uninstall, but does not attribute it to a campaign.